sdk/include/reactos/undocshell.h

```diff
@@ -705,6 +705,25 @@ RealShellExecuteExW(
     _Out_opt_ PHANDLE lphProcess,
     _In_ DWORD dwFlags);
 
+VOID WINAPI
+ShellExec_RunDLL(
```

Comment:
ShellExec_RunDLL and ShellExec_RunDLLA are identical, see comment above.

Reply:
Fixed.